### ISSUE TITLE
Add support for localizations

### ISSUE THREAD

**Issue by jonathontoon** [jonathontoon/tweet-tray#30]
Going to use [react-localization](https://github.com/stefalda/react-localization). Even though Twitter accounts have a language setting which comes down from the API I think basing the locale off what the computer is using will be the way to go. Will avoid allowing a user to customize their language for now.

Below is a list of all the custom strings supported by Tweet Tray (which cannot already be sourced from Twitter's apps), and a list of which languages require localization. 

If you would like to contribute, you can submit a pull request (following these (guidelines)[https://github.com/jonathontoon/tweet-tray#development]) for strings per language found under `app/localizations`. If the there is not a file for your language (e.g: `en-US.json`) you can create it as part of the pull request.

| Language | Status |
| ------------- |:-------------:|
| Arabic (ar) | ❌ |
| Bengali (bn) | ❌ |
| Czech (cz) | ❌ |
| Danish (da) | ❌ |
| German (de) | ✔️ |
| Greek (el) | ❌ |
| Spanish (es) | ✔️ | 
| Persian (pr) | ❌ |
| English (en) | ✔️ |
| Finnish (fi) | ❌ |
| Filipino (fill) | ❌ |
| French (fr) | ✔️ |
| Hebrew (he) | ❌ |
| Hindi (hi) | ❌ |
| Hungarian (hu) | ❌ |
| Indonesian (id) | ❌ |
| Italian (it) | ❌ |
| Japanese (ja) | ✔️ |
| Korean (ko) | ✔️ |
| Malay (msa) | ❌ |
| Dutch (nl) | ❌ |
| Norwegian (no) | ❌ |
| Polish (pl) | ❌ |
| Portuguese (pt) | ❌ |
| Romanian (ro) | ❌ |
| Russian (ru) | ❌ |
| Swedish (sv) | ✔️ |
| Thai (th) | ❌ |
| Turkish (tr) | ❌ |
| Ukrainian (uk) | ❌ |
| Urdu (ur) | ❌ |
| Vietnamese (vi) | ❌ |
| Chinese (Simplified) (zh-CN) | ✔️ |
| Chinese (Traditional) (zh-TW) | ✔️ |

**Comment by harleo** [comment]
I'd be happy to work on German.

**Comment by neauoire** [comment]
Could you link the english localisation file so we can PR it?

**Comment by jonathontoon** [comment]
@neauoire yup, will add it to develop soon. Can support both ways as I'd like to give the opportunity for everyone to contribute, even those who aren't comfortable in committing files/making a PR.

**Comment by neauoire** [comment]
Sounds good I'll try to do french & [Lietal](http://wiki.xxiivv.com/#devine+lu+linvega)

**Comment by maroun-baydoun** [comment]
I can help with Arabic translation. 
Does the UI support RTL at the moment?

**Comment by jonathontoon** [comment]
@neauoire you can now find the strings under `app/localizations` in develop. 

@maroun-baydoun  I haven't tested RTL yet, but getting all of the strings for such languages is the starting place. Once strings are in I will make sure they wont ship without the support. Unless you'd like to give it a go. :)

**Comment by harleo** [comment]
@jonathontoon Done. #115

**Comment by jonathontoon** [comment]
@maroun-baydoun any interest still in doing Arabic?

**Comment by maroun-baydoun** [comment]
@jonathontoon Yes, I will do it and send a PR.

**Comment by riomarmccartney** [comment]
Ja related issue: https://github.com/jonathontoon/tweet-tray/issues/194
